.github/workflows/release.yaml

```diff
@@ -0,0 +1,54 @@
+---
+name: Github Release
```

Comment:
How do we make a release with breaking change? Do we manually update the VERSION 's major version?

Reply:
We can either use `fix!` or `feature!` to indicate a breaking change, or we can force the version with `make initiate-release-as RELEASE_VERSION=<version>`. The former is documented in conventional commits, and the latter is added to the release documentation in the readme.